build/package/scripts/build-typescript.sh

```diff
@@ -56,8 +70,24 @@ if [ -n "${NEXUS_HOST}" ] && [ -n "${NEXUS_USERNAME}" ] && [ -n "${NEXUS_PASSWOR
     npm config set strict-ssl=false
 fi;
 
-echo "Building ..."
+echo "Installing dependencies ..."
 npm ci
+
+echo "Linting ..."
+set +e
+npx eslint src --ext "${LINT_FILE_EXT}" --format compact --max-warnings "${MAX_LINT_WARNINGS}" > eslint-report.txt
+exitcode=$?
+set -e
+
+if [ $exitcode == 0 ]; then
+  echo "OK" > eslint-report.txt
+  copyLintReport
+else
+  copyLintReport
+  exit $exitcode
+fi
```

Comment:
`copyLintReport` is executed in both cases, so it could be placed before the `if`

Reply:
The issue is that this function for the success case needs to run **after** the `echo ...` while for the error case it needs to copy the report **before** exiting. I thought of this to be the only solution to what I wanted to achieve although it is not that beautiful.